internal/obsreportconfig/obsmetrics/obs_receiver.go

```diff
@@ -83,4 +87,40 @@ var (
 		ReceiverPrefix+RefusedLogRecordsKey,
 		"Number of log records that could not be pushed into the pipeline.",
 		stats.UnitDimensionless)
+
+	otelMeter     = global.Meter(MeterName)
+	AttrReceiver  = attribute.Key(ReceiverKey)
+	AttrTransport = attribute.Key(TransportKey)
```

Comment:
Why using the global meter?

Reply:
I've moved those to be created inside of `obsreport.Receiver` using the `ReceiverCreateSettings.MeterProvider`.